tests/README.md

```diff
@@ -0,0 +1,2 @@
+These are smoke tests for riscv64. They are set up to be run via qemu on the
```

Comment:
Move under `tests/riscv64/`?

Reply:
Sorry I don't see the `README.md` in the PR anymore after the latest commit. Is it intended?

Reply:
No :angry: git ate it somehow. Fixed